predicates/auth/jwt_test.go

```diff
@@ -619,6 +619,24 @@ s: JWTPayloadAnyKVRegexp("https://identity.zalando.com/managed-id", "^ssz") -> s
 
 }
 
+func BenchmarkJWTPayloadAnyKV(b *testing.B) {
+	sp := NewJWTPayloadAnyKV()
+	p, err := sp.Create([]interface{}{"https://identity.zalando.com/managed-id", "foo", "iss", "https://identity.zalando.com"})
+	if err != nil {
+		b.Fatalf("Failed to create predicate: %v", err)
+	}
+	benchPredicate(b, p)
+}
+
+func BenchmarkJWTPayloadAllKV(b *testing.B) {
+	sp := NewJWTPayloadAnyKV()
```

Comment:
Shouldn't this be something like "NewJWTPayloadAllKV" here? Or am I missing something here?

Reply:
Haha yes of course!

Reply:
https://github.com/zalando/skipper/pull/3671